strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/LayoutProviderRegistry.java

```diff
@@ -120,51 +123,11 @@ public LayoutProvider addProvider(String alias, LayoutProvider provider)
     {
         LayoutProvider layoutProvider = super.addProvider(alias, provider);
 
-        setRepositoryArtifactCoordinateValidators();
+        configurationManagementService.setRepositoryArtifactCoordinateValidators();
 
         return layoutProvider;
     }
 
-    public void setRepositoryArtifactCoordinateValidators()
```

Comment:
Is this moved somewhere else, or what?

Reply:
It is moved to `ConfigurationManagementServiceImpl`. Only this service may modify `configuration` state in locked mode.

Reply:
Thank you for the explanation! :)